src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -1078,6 +1079,14 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.DisableStandardFrameworkResolution.targets" Condition="'$(DisableStandardFrameworkResolution)' == 'true'" />
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.DesignerSupport.targets" />
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.GenerateAssemblyInfo.targets" Condition="'$(UsingNETSdkDefaults)' == 'true'"/>
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.GenerateImplicitNamespaceImports.targets"
+    Condition="'$(DisableImplicitNamespaceImports)' != 'true'
+      and '$(Language)' == 'C#'
+      and ('$(LangVersion)' == 'preview'
```

Comment:
Not a fan of this but until I can figure out why some tests are using VS's csc.exe which specifies LangVersion=9.0, I'm restricting the LangVersion to >= 10.0 workaround the issue.

Reply:
Anyone know what's the roadmap for C# 10 on Full Framework? I saw some failures on the Full Framework legs where C# 10 isn't available. I'm not sure if that's just because the VS version is out of date or not though.